compiler/env/OMRCompilerEnv.hpp

```diff
@@ -107,6 +107,8 @@ class OMR_EXTENSIBLE CompilerEnv
 
    TR::PersistentAllocator &persistentAllocator() { return _persistentAllocator; }
 
+   OMRPortLibrary * const portLib;
```

Comment:
Who initializes this?

Reply:
Currently it's set to NULL in the constructor. This part is still waiting on the decision for port library initialization.